code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -0,0 +1,429 @@
+	return message
+
+
+//Colossus drop, forces the listeners to obey certain commands
+/obj/item/organ/vocal_cords/colossus
+	name = "divine vocal cords"
+	desc = "They carry the voice of an ancient god."
+	icon_state = "voice_of_god"
+	zone = "mouth"
+	slot = "vocal_cords"
+	actions_types = list(/datum/action/item_action/organ_action/colossus)
+	var/next_command = 0
+	var/cooldown_stun = 900
+	var/cooldown_damage = 600
+	var/cooldown_meme = 300
+	var/cooldown_none = 150
+	var/base_multiplier = 1
+	spans = list("colossus","yell")
+
+/datum/action/item_action/organ_action/colossus
+	name = "Voice of God"
+	var/obj/item/organ/vocal_cords/colossus/cords = null
+
+/datum/action/item_action/organ_action/colossus/New()
+	..()
+	cords = target
+
+/datum/action/item_action/organ_action/colossus/IsAvailable()
+	if(world.time < cords.next_command)
+		return FALSE
+	if(!owner)
+		return FALSE
+	if(!owner.can_speak())
+		return FALSE
+	if(check_flags & AB_CHECK_CONSCIOUS)
+		if(owner.stat)
+			return FALSE
+	return TRUE
+
+/datum/action/item_action/organ_action/colossus/Trigger()
+	. = ..()
+	if(!IsAvailable())
+		if(world.time < cords.next_command)
+			owner << "<span class='notice'>You must wait [(cords.next_command - world.time)/10] seconds before Speaking again.</span>"
+		return
+	var/command = stripped_input(owner, "Speak with the Voice of God", "Command", max_length = 140)
+	if(!command)
+		return
+	owner.say(".x[command]")
+
+/obj/item/organ/vocal_cords/colossus/can_speak_with()
+	if(world.time < next_command)
+		owner << "<span class='notice'>You must wait [(next_command - world.time)/10] seconds before Speaking again.</span>"
+		return FALSE
+	if(!owner)
+		return FALSE
+	if(!owner.can_speak())
+		owner << "<span class='warning'>You are unable to speak!</span>"
+		return FALSE
+	if(owner.stat)
+		return FALSE
+	return TRUE
+
+/obj/item/organ/vocal_cords/colossus/speak_with(message)
+	var/spoken = uppertext(message)
+	playsound(get_turf(owner), 'sound/magic/clockwork/invoke_general.ogg', 300, 1, 5)
+
+	var/mob/living/list/listeners = list()
+	for(var/mob/living/L in get_hearers_in_view(8, owner))
+		if(!L.ear_deaf && L != owner && L.stat != DEAD)
+			listeners += L
+
+	if(!listeners.len)
+		next_command = world.time + cooldown_none
+		return
+
+	var/power_multiplier = base_multiplier
+	spans = list("colossus","yell") //reset spans, just in case someone gets deculted or the cords change owner
+
+	if(owner.mind)
+		//Chaplains are very good at speaking with the voice of god
+		if(owner.mind.assigned_role == "Chaplain")
+			power_multiplier *= 2
+		//Command staff has authority
+		if(owner.mind.assigned_role in command_positions)
+			power_multiplier *= 1.4
+		//Why are you speaking
+		if(owner.mind.assigned_role == "Mime")
+			power_multiplier *= 0.5
+
+	//Cultists are closer to their gods and are more powerful, but they'll give themselves away
+	if(iscultist(owner))
+		power_multiplier *= 2
+		spans = list("narsie")
```

Comment:
Size 5 is more than large enough, I really don't want people able to talk in a size 15 font even if they ARE antags.